src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -741,6 +740,22 @@ public class SimpleEvents {
 				)
 				.since("2.7");
 
+		Skript.registerEvent("Unknown Command", SimpleEvent.class, UnknownCommandEvent.class, "[player] (wrong|unknown) (cmd|command) [send|use]")
+			.description("Called when the player uses a command that does not exist.")
+			.examples(
+				"on unknown command:",
+				"\tset event-unknown command message to \"Hey, this command does not exist.\""
+			)
+			.since("INSERT VERSION");
+
+		Skript.registerEvent("Player Item Cooldown", SimpleEvent.class, PlayerItemCooldownEvent.class, "[player] item cool[ ]down")
+			.description("Called when a player's item cooldown changes.", " ","You must at least wait 1 tick if you need the cooldown or check it", "Because the cooldown won't have been set on the item itself when the event is called in case something wants to cancel or modify the cooldown.")
```

Comment:
how should i

Reply:
and should i just have event-integer in the BukkitEventValues

Reply:
by editing the getter and changer in the expression, there are examples in things like ExprExperience, for example.
And no, I would not provide event-integer. I would provide event-timespan, though.

Reply:
ExprItemCooldown doesn't look like ExprExperience class :/

Reply:
![image](https://github.com/SkriptLang/Skript/assets/125564466/3d98e26f-7157-478e-b32f-d0b5322b448a)
![image](https://github.com/SkriptLang/Skript/assets/125564466/81b6ad6e-4598-4cd5-9604-86a0349242a5)

Reply:
Of course it doesn't. The point you need to look at is checking the instance of the event. It's up to you to implement the actual change in the logic when it is the cooldown change event.

Reply:
actually i found that setting it in the event will keep setting it forever, because if i set that it calls the event

Reply:
yes that's exactly why the expression needs a changer that modifies the event's values instead of using setCooldown()
You may also want to prevent setting the cooldown after a delay in the event, but that's optional.

Reply:
but for modifing the expression, how should i change that since it uses setCooldown(), do i add a checker to check if event instanceof PlayerItemCooldownEvent?

Reply:
and do other things in that
and if its not it does its regular thing? or thats a bad idea

Reply:
yes that's exactly why I told you to look at ExprExperience